### PR TITLE
make reclassify value public in the GIS package

### PR DESCRIPTION
**Pull request by Gustry**
### What does it fix?
* Ticket: partial fix for #3469
* Funded by: DFAT
* Description: make reclassify value public in the GIS package

### Checklist:
- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [ ] Request someone to review or test your PR

### CODE REVIEW

**Comment by Gustry** [comment]
@ismailsunni I have included the function to retrieve the geometry type using the QGIS API. I'm using it in the impact function. Let me know if it's ok.

File: safe/gis/tools.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+from safe.definitions.layer_geometry import (
+    layer_geometry_raster,
+    layer_geometry_point,
+    layer_geometry_line,
+    layer_geometry_polygon,
+)
+from safe.utilities.gis import is_raster_layer, is_point_layer, is_line_layer
+
+
+__copyright__ = "Copyright 2016, The InaSAFE Project"
```
Comment (ismailsunni) [inline safe/gis/tools.py]: 2017

File: safe/gis/tools.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+def reclassify_value(one_value, ranges):
+    """This function will return the classified value according to ranges.
+
+    The algorithm will return None if the continuous value has not any class.
+
+    :param one_value: The continuous value to classify.
+    :type one_value: float
+
+    :param ranges: Classes,
```
Comment (ismailsunni) [inline safe/gis/tools.py]: comma ?

File: safe/gis/tools.py (hunk 1 above)
Comment (Gustry) [reply safe/gis/tools.py]: I guess we missed a lot since January :)

**Comment by Gustry** [comment]
Both fixed, thanks

**Pull request merged**